Bottom: 52070af12d011fdc659b828cb5c7689796a3e1a9
Top:    f49b46ce265a0adae2faf1f0f1b90c51f752861e
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-12 17:19:01 +0800

Refresh of autofs-5.0.7-fix-probe-each-nfs-version-in-turn-for-singleton-mounts.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 25179b1..0242c11 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -50,6 +50,7 @@
 - fix fix map entry duplicate offset detection.
 - probe each nfs version in turn for singleton mounts.
 - add changlog entry for coverity fixes.
+- fix probe each nfs version in turn for singleton mounts.
 
 25/07/2012 autofs-5.0.7
 =======================
